resources/Microsoft.Windows.Assertion/Microsoft.Windows.Assertion.psd1

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+@{
+    RootModule           = 'Microsoft.Windows.Assertion.psm1'
+    ModuleVersion        = '0.1.0'
+    GUID                 = 'e3510ba2-cc19-4fb2-872a-a40833c30e58'
+    Author               = 'Microsoft Corporation'
+    CompanyName          = 'Microsoft Corporation'
+    Copyright            = '(c) Microsoft Corp. All rights reserved.'
+    Description          = 'DSC Module for ensuring the system meets certain specifications'
+    PowerShellVersion    = '7.2'
+    DscResourcesToExport = @(
+        'OsEditionId',
+        'SystemArchitecture',
+        'ProcessorArchitecture',
+        'HyperVisorPresent',
+        'OsInstallDate',
+        'OsVersion',
+        'CsManufacturer',
+        'CsModel',
+        'CsDomain',
+        'PowerShellVersion',
+        'PnPDevice'
+    )
+    PrivateData          = @{
+        PSData = @{
+            # Tags applied to this module. These help with module discovery in online galleries.
```

Comment:
niy: Can you both add `LicenseUri` and `ProjectUri`? This helps user discover the project when they're installing it from PowerShell Gallery :)